.github/workflows/release.yml

```diff
@@ -88,21 +88,21 @@ jobs:
             - name: Push to npm
               run: |
                   yarn npm-publish
-              env:
-                  DEVCYCLE_PROD_SLEUTH_API_TOKEN: ${{ secrets.SLEUTH_API_KEY }}
               if: inputs.draft != true
 
             - name: Push to npm (dry run)
               run: |
                   yarn npm-publish --dry-run
-              env:
-                  DEVCYCLE_PROD_SLEUTH_API_TOKEN: ${{ secrets.SLEUTH_API_KEY }}
               if: inputs.draft == true
 
             - name: Build UMD bundles
               run: |
                   yarn nx build:cdn js
 
             - name: Upload CDN Release
-              run: aws s3 sync dist/sdk/cdn/js s3://js.devcycle.com/ --include "devcycle.min.js*" --acl public-read
+              run: |
+                JQ_PATH=".version"
```

Comment:
doesn't really seem like this needs to be assigned to a variable and could just be written directly on the next line